src/components/project/sections/largeTextArea.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  colors,
+  dimensions,
+  breakpoints,
+} from "../../../../src/styles/variables";
+import styled from "@emotion/styled";
+import { getRemSize } from "../../../../src/styles/globalCss";
+import { Row } from "../../global/grid/Row";
+import { Col } from "../../global/grid/Col";
+
+interface IProps {
+  text: string;
+}
+
+const StyledTitleRow = styled(Row)`
+  margin: 120px 0 120px 0;
+
+  @media (max-width: ${breakpoints.md}px) {
+    margin: 120px 0 120px 0;
+  }
+`;
+
+const StyledHeading = styled.h1`
```

Comment:
There should only ever be 1 `h1` element on a page at the top. So this element would be a h2
```suggestion
const StyledHeading = styled.h2`
```